core/input/input.h

```diff
@@ -158,6 +156,19 @@ class Input : public Object {
 
 	HashMap<int, VibrationInfo> joy_vibration;
 
+	struct TouchpadFingerInfo {
+		Vector2 position = Vector2();
+		float pressure = 0.0f;
+	};
+
+	struct TouchpadInfo {
+		int num_touchpads = 0;
+		// The first int index refers to touchpad ID, the second one refers to finger ID on that touchpad.
+		HashMap<int, HashMap<int, TouchpadFingerInfo>> touchpad_fingers;
```

Comment:
It probably makes sense for it to be
```suggestion
		HashMap<Vector2i, TouchpadFingerInfo> finger_info;
```
Alongside with `Vector<int> fingers`